.devcontainer/docker-compose.yml

```
@@ -11,6 +11,7 @@ services:
     container_name: nyu
     volumes:
       - ..:/app
+      - ~/.bluemix/apikey.json:/home/devops/apikey.json
```

```suggestion
      - ../.vscode/apikey.json:/home/devops/apikey.json
```
Because you put the `apikey.json` in the folder `.vscode` and we need share the same key, I think it should be changed.

---

Oh yes, sorry about that. The code is correct. The problem is that I shouldn't put apikey in the vscode folder. That was a mistake. You should put your own apikey.json file in the ~/.bluemix folder in your PC to run the script. I'll change the code. Thanks for the comments.